README.rst

```diff
@@ -146,6 +150,16 @@ Conda (Anaconda)
 
 Currently, it supports only python version 2.7, 3.6 and 3.7.
 
+Snap (Snapcraft)
+----------------
+
+Download the latest ``dvc_*.snap`` from the
```

Comment:
Let's add a note that we are waiting for snap approval. And let's include a link and ask people to comment under it to speed that up, maybe that would put some pressure on snap guys to review our request 😄

Reply:
indeed. I find myself constantly updating the checklist in this PR.